### PR TITLE
ibv_xsrq_pingpong maximum allowed number of -n option value is 8192

### PR DESCRIPTION
**Pull request by PavamanSubramaniyam**
Any number beyond this range will fail

Please use value less than or equal to 8192 for your test

Signed-off-by: Pavaman Subramaniyam <pavsubra@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by PavamanSubramaniyam** [comment]
I have executed the tests with the changes made in the yaml file for -n option and pasting the results here.

# avocado run --test-runner runner ib_pingpong.py -m ib_pingpong.py.data/ib_pingpong_roce.yaml
JOB ID     : 93b01e56655173a2b67a5b7e37295a5e379707ac
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2022-11-02T01.20-93b01e5/job.log
 (1/9) ib_pingpong.py:PingPong.test_ib_pingpong;run-Test-xsrq_pingpong-mtu-1500-parameters-898e: PASS (36.16 s)
 (2/9) ib_pingpong.py:PingPong.test_ib_pingpong;run-Test-xsrq_pingpong-mtu-2000-parameters-31c6: PASS (36.87 s)
 (3/9) ib_pingpong.py:PingPong.test_ib_pingpong;run-Test-xsrq_pingpong-mtu-3000-parameters-2f1c: PASS (37.06 s)
 (4/9) ib_pingpong.py:PingPong.test_ib_pingpong;run-Test-xsrq_pingpong-mtu-4000-parameters-e13f: PASS (37.15 s)
 (5/9) ib_pingpong.py:PingPong.test_ib_pingpong;run-Test-xsrq_pingpong-mtu-5000-parameters-787d: PASS (36.95 s)
 (6/9) ib_pingpong.py:PingPong.test_ib_pingpong;run-Test-xsrq_pingpong-mtu-6000-parameters-9bdc: PASS (36.88 s)
 (7/9) ib_pingpong.py:PingPong.test_ib_pingpong;run-Test-xsrq_pingpong-mtu-7000-parameters-b002: PASS (37.07 s)
 (8/9) ib_pingpong.py:PingPong.test_ib_pingpong;run-Test-xsrq_pingpong-mtu-8000-parameters-c721: PASS (37.09 s)
 (9/9) ib_pingpong.py:PingPong.test_ib_pingpong;run-Test-xsrq_pingpong-mtu-9000-parameters-a8c1: PASS (37.15 s)
RESULTS    : PASS 9 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2022-11-02T01.20-93b01e5/results.html
JOB TIME   : 333.39 s

File: io/net/infiniband/ib_pingpong.py.data/ib_pingpong_roce.yaml [hunk 1]
```diff
@@ -17,8 +17,8 @@ Test: !mux
         ext_test_opt: -s 524288,-s 524288 -n 10000,-s 524288 -n 100000,-q 32,-e -s 524288 -n 10000 -q 32
     xsrq_pingpong:
         tool: ibv_xsrq_pingpong
-        test_opt: -s 1024,basic,-n 100000,-e,-l 2,-m 1024
-        ext_test_opt: -s 524288,-s 524288 -n 10000,-s 524288 -n 100000,-e -s 524288 -n 10000
+        test_opt: -s 1024,basic,-n 8192,-e,-l 2,-m 1024
+        ext_test_opt: -s 524288,-s 524288 -n 8192,-s 524288 -n 8192,-e -s 524288 -n 8192
```
Comment (Naresh-ibm) [inline io/net/infiniband/ib_pingpong.py.data/ib_pingpong_roce.yaml:21]: @PavamanSubramaniyam was this failing before ?
any recent changes that causing this fail for beyond this value?

Comment (PavamanSubramaniyam) [reply io/net/infiniband/ib_pingpong.py.data/ib_pingpong_roce.yaml:21]: @Naresh-ibm Yes, it was failing before. So, had reported a bug and the developer mentioned that for ibv_xsrq_pingpong utility the maximum allowed number of -n option value is 8192.
So, accordingly changing the same in the yaml file.

**Review by abdhaleegit — APPROVED** [review]
LGTM

**Review by Naresh-ibm — APPROVED** [review]
Thank you @abdhaleegit and @PavamanSubramaniyam 
LGTM

**Pull request merged**